packedForest/src/forestTypes/basicForests/baseUnprocessedNode.h

```diff
@@ -78,9 +78,7 @@ namespace fp{
 				}
 
 				inline std::vector<int> returnOutSampleVec(){
-					 std::vector<int> out = obsIndices->returnOutSample();
-					 //out = obsIndices->returnOutSample();
-					 return out;
+					 return obsIndices->returnOutSample();
```

Comment:
this returns.... the indices of the outsamples in the tree?  or in `x`?

Reply:
It returns the out-of-bag indices in the leaf node.  I've changed the function name accordingly.